share/spice/amazon/branded_products_item.handlebars

```diff
@@ -0,0 +1,27 @@
+<div class="tile  tile--pr tile--c has-detail  tile--{{parentId}} opt--t-xxs" data-link="{{url}}">
```

Comment:
For the purpose of the experiment, I've simply copied this from the open-source template to allow for easily adding the new Brand link when needed.

If the test is a success, the Products template should be updated to allow for the Brand link

Reply:
👍 Nice approach, much cleaner than adding IF then THIS etc to the other template.